crates/cxx-qt-lib/src/core/qpoint.rs

```diff
@@ -41,6 +41,21 @@ mod ffi {
         #[doc(hidden)]
         #[rust_name = "qpoint_to_qstring"]
         fn toQString(value: &QPoint) -> QString;
+        #[doc(hidden)]
+        #[rust_name = "qpoint_plus"]
+        fn operatorPlus(a: &QPoint, b: &QPoint) -> QPoint;
+        #[doc(hidden)]
+        #[rust_name = "qpoint_minus"]
+        fn operatorMinus(a: &QPoint, b: &QPoint) -> QPoint;
+        #[doc(hidden)]
+        #[rust_name = "qpoint_mul_f32"]
+        fn operatorMul(a: f32, b: &QPoint) -> QPoint;
+        #[doc(hidden)]
+        #[rust_name = "qpoint_mul_f64"]
+        fn operatorMul(a: f64, b: &QPoint) -> QPoint;
+        #[doc(hidden)]
+        #[rust_name = "qpoint_mul_i32"]
+        fn operatorMul(a: i32, b: &QPoint) -> QPoint;
```

Comment:
has divide been missed for QPoint?

Reply:
Yes, I didn't see that there was Qt API for this that rounds the results.